server/command.go

```diff
@@ -21,14 +21,19 @@ func (p *Plugin) createCommand(syncLinkedChannels bool) *model.Command {
 		p.API.LogWarn("Unable to get the MS Teams icon for the slash command")
 	}
 
+	autoCompleteData := getAutocompleteData(syncLinkedChannels)
```

Comment:
offtopic: Every time we added a command we had to update a string with the new command, which is not ideal, but as a bonus we had to do it twice 😅 The other part of this change is marked by a ⭐!